### PR TITLE
chore(i18n): update Simplified Chinese localization

### PR DESCRIPTION
**Pull request by zhen-shi**
1. Proofread and optimize existing Simplified Chinese translation:

	- Align fediverse proper nouns with [Mastodon Community Translation](https://crowdin.com/project/mastodon/zh-CN), including following, follower, status, pin, favourite, boost, and so on.

	- Optimize descriptions, fix grammatical errors.

	- Enhance error messages to be more user-friendly.

2. Add Simplified Chinese translation of Streaming and AI feature.

### CODE REVIEW

File: IceCubesApp/Resources/Localization/Localizable.xcstrings [hunk 1]
```diff
@@ -23891,8 +23891,14 @@
...
           }
+        },
+        "zh-Hans" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "禁用以隐藏AI辅助工具选项，例如使用AI生成文案和图片描述文本。内容使用 OpenAI API 生成。如需了解更多信息，请参阅我们的隐私政策。"
+          }
         }
-      }
+      }
```
Comment (Kyle-Ye) [inline IceCubesApp/Resources/Localization/Localizable.xcstrings:23901]: Nit: An extra unexpected space here.

```suggestion
      }
```

**Review by Kyle-Ye — APPROVED** [review]
LGTM.

**Comment by Jerry23011** [comment]
LGTM

What do you think CC @nixzhu

File: IceCubesApp/Resources/Localization/Localizable.xcstrings [hunk 2]
```diff
@@ -39452,7 +39464,7 @@
         "zh-Hans" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "已转发"
+            "value" : "转发了你的嘟文"
```
Comment (nixzhu) [inline IceCubesApp/Resources/Localization/Localizable.xcstrings:39467]: 转嘟了你的嘟文

**Review by Jerry23011 — APPROVED** [review]

**Comment by nixzhu** [comment]
LGTM

**Review by nixzhu — APPROVED** [review]

**Pull request merged**